cylc/flow/broadcast_mgr.py

```diff
@@ -340,6 +348,7 @@ def put_broadcast(
                                 namespace,
                                 coerced_setting,
                             ):
+                                bad_settings.append(setting)
```

Comment:
@wxtim I have rebased this onto master which now contains your recent change to detect incompatible `[remote]host` / `platform` broadcasts.

This line will report the setting back to the client. I've extended the new tests in `test_broadcast_mgr` to assert this.